Dockerfile

```diff
@@ -0,0 +1,18 @@
+# java
+#
+# VERSION       Java 8
+FROM centos
```

Comment:
was there a conscious decision to use this image instead of a Java one? Similarly, why did we choose CentOS over other distros?